packages/base/src/3dview/mainview.tsx

```diff
@@ -305,7 +307,8 @@ export class MainView extends React.Component<IProps, IStates> {
           color: 'black',
           opacity: 0.2,
           transparent: true,
-          side: THREE.DoubleSide
+          side: THREE.DoubleSide,
+          wireframe: this.state.wireframe
```

Comment:
Also not on the clipplane control object

```suggestion
          side: THREE.DoubleSide
```